python/triton/language/core.py

```diff
@@ -778,12 +778,26 @@ def T(self):
         assert False, "Transposition must be created by the AST Visitor"
 
     @builtin
-    def to(self, dtype, bitcast=False, _builder=None):
+    def to(self, dtype, fp_downcast_rounding: str = None, bitcast=False, _builder=None):
+        """
+        Casts the tensor to the given :code:`dtype`.
+        :param dtype: The target data type.
+        :type dtype: DType
+        :param fp_downcast_rounding: The rounding mode for downcasting floating-point values. \
```

Comment:
Thanks for writing documentation <3

I don't think you need to end lines with backslashes here, indentation is sufficient.